javascript-light-bulb/main.js

```diff
@@ -0,0 +1,14 @@
+var $lightbulb = document.querySelector('button');
+var $htmlBackground = document.querySelector('html');
+
+var offAndOnFunction = offAndOn;
+$lightbulb.addEventListener('click', offAndOnFunction);
+function offAndOn(event) {
+  if (($lightbulb.className === 'yellow-lightbulb') && ($htmlBackground.className === 'white')) {
```

Comment:
Instead of making your functionality reliant on the element having a specific class, I would create a variable to keep track of if the lightbulb is on or not; its value should be a boolean.